src/functions/editor.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * Check if the current theme is a block theme.
+ *
+ * @since TBD
+ *
+ * @return bool Whether the current theme is a block theme supporting full-site editing.
+ */
+if ( ! function_exists( 'tec_is_full_site_editor' ) ) {
+	function tec_is_full_site_editor() {
```

Comment:
To be correctly parsed by the documentation generators and IDEs, the doc-block should be moved before the definition of the `tec_is_sull_site_editor` function, not before the check to see if it's defined already.